apps/prairielearn/src/pages/clientFilesQuestion/clientFilesQuestion.js

```diff
@@ -35,7 +35,15 @@ module.exports = function (options = { publicEndpoint: false }) {
       }
 
       const question_course = await getQuestionCourse(res.locals.question, res.locals.course);
-      const coursePath = chunks.getRuntimeDirectoryForCourse(question_course);
+      if (question_course.path == null) {
```

Comment:
This seems like the kind of thing we could add a `NOT NULL` constraint for, right?

Reply:
Yes, this is not the first time this column has caused me some grief...

Reply:
I will wait until #9257 is merged before merging this one, so that this check can be removed.